apps/patient/src/components/PharmacyFilters.tsx

```diff
@@ -37,8 +44,22 @@ export const PharmacyFilters = ({
           setEnable24Hr(!enable24Hr);
         }}
       >
-        Open 24 Hours
+        24 Hours
       </Button>
+      {paymentMethod === 'Cash Price' ? (
+        <Button
+          size="sm"
+          borderRadius="lg"
+          variant="filter"
+          isActive={enablePrice}
+          onClick={() => {
+            if (setEnablePrice) setEnablePrice(false);
+            setEnablePrice(!enablePrice);
+          }}
+        >
+          Price
```

Comment:
(throw away comment) I thought we were going to consider calling this "coupon" at some point?

Reply:
oh we do call it a coupon on the status page, but upfront it's simplified to just a price filter